qp_klp/Step.py

```diff
@@ -327,12 +327,13 @@ def _quality_control(self, config, input_file_path):
                            self.master_qiita_job_id,
                            config['job_max_array_length'],
                            config['known_adapters_path'],
+                           config['movi_executable_path'],
```

Comment:
just out of curiosity, why this change?

Reply:
This goes back to a discussion we had with Amanda on the mg-scripts PR associated with this one. The group felt that default constructor parameters were not very desirable. I generally agree with that notion but I liked adding new parameters with defaults in order to preserve legacy behavior. Since I knew this was the sentiment I used the opportunity of adding a new parameter to go back and make the others positional as well.